src/validate/project.rs

```diff
@@ -11,6 +11,8 @@ pub struct Project {
     tags: HashSet<String>,
     /// Projects must have a maintainer at Embark
     pub maintainers: eyre::Result<HashSet<String>>,
+    /// Projects must not be archived
+    archive_status: eyre::Result<()>,
```

Comment:
I'm not sure failing CI in this repository is terribly useful, unless people are receiving notifications for every failed build (I know I'm not.) I think we'd want something more attention grabbing, such as creating an issue or notifying people in slack to update the open source website.

We'll likely only notice that CI is failing when we're adding a new feature in this repository, and at that point it's just busy work before whatever that feature is can be merged, regardless of whether that feature is dependent on not archived projects or not.

I'm also not sure what the issue is with archiving that it must not be archived? Surely we'd want to archive repositories that we no longer intent on maintaining?

Reply:
We have this running on a cron job that sends a message to slack with any problems, so this is already done 👍 

> I'm also not sure what the issue is with archiving that it must not be archived? Surely we'd want to archive repositories that we no longer intent on maintaining?

This script works off the list of officially maintained projects. When a project is no longer maintained it is removed from the list and archived from GitHub, so the script doesn't check them.

Reply:
Here is an example https://embark-studios.slack.com/archives/CRAG3JXC1/p1608031161112200

Reply:
> This script works off the list of officially maintained projects. When a project is no longer maintained it is removed from the list and archived from GitHub, so the script doesn't check them.

Right so this is just checking that this invariant is still being upheld? I would update the comments then so that includes information about that, because right now that's not really conveyed in the code or in the comments.

Reply:
Yes that's right. The criteria for which projects are analysed by which command is documented here: https://github.com/EmbarkStudios/opensource/blob/main/content/tooling.md#commands

Reply:
Ah, this comment was out of date though https://github.com/EmbarkStudios/opensource/pull/29/commits/5010e4149c38622f8ee5411f3bfc1c4f16d15a6e